mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -92,21 +108,23 @@ CLASS aij net/minecraft/entity/Entity
 		ARG 1 cursorDeltaX
 		ARG 3 cursorDeltaY
 	METHOD a requestTeleport (DDD)V
-		ARG 1 x
-		ARG 3 y
-		ARG 5 z
+		ARG 1 destX
+		ARG 3 destY
+		ARG 5 destZ
 	METHOD a setPositionAnglesAndUpdate (DDDFF)V
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
 		ARG 7 yaw
 		ARG 8 pitch
-	METHOD a setPositionAndRotations (DDDFFIZ)V
+	METHOD a updateClientPositionAndAngles (DDDFFIZ)V
```

Comment:
This could make specific reference to being related to tracking -- the current name doesn't make that immediately clear. Not sure of alternative, though. Possibly something like `updateTrackedPositionAndAngles`?

Reply:
Sure, `tracked` instead of `client` works for me. The intention was to make it clear this method only exists on the client, but you can probably extend it to anything in the `tracked` category.